content/docs/1.9.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

```diff
@@ -77,10 +77,14 @@ The Longhorn system backup offers the following volume backup policies:
 
 #### Create a System Backup Using the Longhorn UI
 
-1. Go to the `System Backup` page in the `Setting` drop-down list.
+1. Go to the `System Backup` screen in the `Setting` drop-down list.
+
```

Comment:
I don't understand why we needed extra blank lines between the numbering points. I believe we usually avoid adding extra line breaks because we follow the Markdown [ordered lists](https://www.markdownguide.org/basic-syntax/#ordered-lists) format, which prevents the numbering from breaking. However, even with a line break, it doesn’t seem to affect the formatting on the website. So, is the suggested change for internal codebase management?